ProcessLib/HT/HTProcessData.h

```diff
@@ -91,20 +89,17 @@ struct HTProcessData
     //! Assignments are not needed.
     void operator=(HTProcessData&&) = delete;
 
-    Parameter<double> const& porosity;
-    Parameter<double> const& intrinsic_permeability;
-    Parameter<double> const& specific_storage;
-    Parameter<double> const& viscosity;
+    PorousMediaProperties porous_media_properties;
+    std::unique_ptr<MaterialLib::Fluid::FluidProperty> viscosity_model;
```

Comment:
You can replace
```
std::unique_ptr<MaterialLib::Fluid::FluidProperty> viscosity_model;
std::unique_ptr<MaterialLib::Fluid::FluidProperty> fluid_density;
Parameter<double> const& specific_heat_capacity_fluid;
Parameter<double> const& thermal_conductivity_fluid;
```
with 
```
const std::unique_ptr<MaterialLib::Fluid::FluidProperties>  _fluid_properties;
```
There is a creator for ``MaterialLib::Fluid::FluidProperties`` as
```
std::unique_ptr<FluidProperties> createFluidProperties(
    BaseLib::ConfigTree const& config);
```